ekg-prometheus-adapter.cabal

```diff
@@ -17,7 +17,7 @@ library
   hs-source-dirs:      src
   exposed-modules:     System.Remote.Monitoring.Prometheus
   build-depends:       base >= 4.7 && < 5
-                     , prometheus < 0.5.0
+                     , prometheus < 2.2
```

Comment:
Do you know if your changes here require particular new version of prometheus?  If so, it might be good to add lower bounds.